logical/src/main/java/org/apache/drill/common/logical/data/visitors/LogicalVisitor.java

```diff
@@ -63,4 +65,7 @@
     public RETURN visitUnion(Union union, EXTRA value) throws EXCEP;
     public RETURN visitWindow(Window window, EXTRA value) throws EXCEP;
     public RETURN visitWriter(Writer writer, EXTRA value) throws EXCEP;
+
+    public RETURN visitUnnest(Unnest unnest, EXTRA value) throws EXCEP;
+    public RETURN visitLateralJoin(LateralJoin lateralJoin, EXTRA value) throws EXCEP;
```

Comment:
We might not have 'Join' in place of LateralJoin during parsing stage ( I am experimenting a different logical node other than 'Join, to avoid having Join related rules kick in since they are not needed.